###### 10 a. Which conditional distribution do these statistics refer to, Y given X, or X given Y ?

Y:Race of victim

X Race of murderer 

Of all blacks slain (Y) in 2005, 91% were slain by blacks (X) and of all whites slain in 2005 (Y), 83% were slain by whites (X).

These statistics refer to the conditional distribution: X given Y. 

###### 10 b. Calculate and interpret the odds ratio between X and Y .

In [3]:
b1<-.91
b2<- 1-b1
w1<- .83
w2<-1-w1
CTable_10<-array(data=c(b1,w1,b2,w2),dim=c(2,2),
dimnames=list(Murderer=c('Black','White'),
Victim=c("B","W")))

#Odds_b<-b1/b2; Odds_b
#Odds_w<-w1/w2; Odds_w
#OR_b_w<-Odds_b/Odds_w; OR_b_w

OR.hat10<-CTable_10[1,1]*CTable_10[2,2] / (CTable_10[2,1]*CTable_10[1,2])
OR.hat10

[1] 2.07095

Odds Ratio Interpretation: Murderers who are Blacks are 2.07 times more likely to slain their own ethnicity than murders who are whites in 2005.

###### 10 c. Given that a murderer was white, can you estimate the probability that the victim was white? What additional information would you need to do this? (Hint: How could you use Bayes’s Theorem?)
V=Victim

M=Murderer

P(V=White|M=White)= $\frac{P(M=White|V=White)(P(V=White))}{P(M=White)}$

We already have the following information:

P(M=White|V=White)= .83

P(V=White)= 0.09+ 0.17=0.26

We still need the probability that the murderer is P(M)=White in order to calculate P(V=White|M=White). 